pkg/controller/configurationpolicy/configurationpolicy_controller.go

```diff
@@ -187,6 +187,8 @@ func (r *ReconcileConfigurationPolicy) Reconcile(request reconcile.Request) (rec
 // PeriodicallyExecConfigPolicies loops through all configurationpolicies in the target namespace and triggers
 // template handling for each one. This function drives all the work the configuration policy controller does.
 func PeriodicallyExecConfigPolicies(freq uint, test bool) {
+	cachedApiResourceList := []*metav1.APIResourceList{}
+	cachedApiGroupsList := []*restmapper.APIGroupResources{}
```

Comment:
I don't think it really matters because the resource list is only used inside the loop below where those variables are declared, but I can change it to a global if that makes the code more readable